htdocs/core/actions_massactions.inc.php

```diff
@@ -1742,7 +1742,7 @@
 				// If no SQL error, we redirect to the request form
 				if (!$error) {
 					// Calculate number of days consumed
-					$nbopenedday = num_open_day($objecttmp->date_debut_gmt, $objecttmp->date_fin_gmt, 0, 1, $objecttmp->halfday);
+					$nbopenedday = num_open_day($objecttmp->date_debut_gmt, $objecttmp->date_fin_gmt, 0, 1, $objecttmp->halfday, '', $objecttmp->fk_user);
```

Comment:
A more efficient solution would be to propagate into existing field country the $objecttmp->country_id  of the user.
I have modified the method num_open_day() so we can give directly the country id when we have it.